opentelemetry-api/java-agent/extensions/CustomizeLogRecordExporter/src/main/java/com/example/DemoAutoConfigurationCustomizerProvider.java

```diff
@@ -0,0 +1,44 @@
+package com.example;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizer;
+import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizerProvider;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.logs.export.LogRecordExporter;
+import io.opentelemetry.sdk.logs.export.SimpleLogRecordProcessor;
+import io.opentelemetry.sdk.trace.SdkTracerProviderBuilder;
+import io.opentelemetry.sdk.trace.SpanLimits;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This is one of the main entry points for Instrumentation Agent's customizations. It allows
+ * configuring the {@link AutoConfigurationCustomizer}. See the {@link
+ * #customize(AutoConfigurationCustomizer)} method below.
+ *
+ * <p>Also see https://github.com/open-telemetry/opentelemetry-java/issues/2022
+ *
+ * @see AutoConfigurationCustomizerProvider
+ */
+@AutoService(AutoConfigurationCustomizerProvider.class)
+public class DemoAutoConfigurationCustomizerProvider
+    implements AutoConfigurationCustomizerProvider {
+
+    @Override
+    public void customize(AutoConfigurationCustomizer autoConfiguration) {
+
```

Comment:
if you add a `System.out.println` here, does it get called?

Reply:
no